ipe_process_orchestrator/orchestrator.py

```diff
@@ -18,25 +18,28 @@ def __init__(self, props, original_df, api_handler) -> None:
         """
         Initialize the orchestrator
         """
-        self.orginal_df: pd.DataFrame = original_df
+        self.original_df: pd.DataFrame = original_df
         self.props = props
         self.api_handler: APIHandler = api_handler
         self.filter_df_course_ids = pd.DataFrame()
 
-    def _clean_up_ipe_dataframe(self) -> None:
+    def filter_course_list_to_run_and_cleanup(self) -> None:
         """
-        Clean up the dataframe
-        1. leading and trailing spaces df.columns
-        2. only courses id list with values that are numbers. Removes Shell, shell, empty, n/a, shell(23333)
-        3. the original dataframe will remain same and the filtered dataframe will be created with courseIds
-
+        1. this filter the courses that need to be run based on the 'Script Run?' and `When does the script run columns
```

Comment:
change 

based on the 'Script Run?' and `When does the script run columns

into 

based on the 'Script Run?' and 'When does script run? (Feb, June, Oct)' columns

Reply:
Done